### DISCUSSION TITLE
Are remants of SFML and Allegro still there?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/eepp#61] [Q&A] [answered]
Or they were completed purged and replaced with SDL2?

https://github.com/SpartanJ/ecode/discussions/215

**Comment by SpartanJ** [comment, marked as answer]
They were purged from the source code, they still can be recovered from the git history, they would work probably with almost no changes, but SDL2 is much superior feature wise I've never found a good reason to keep them. Implementing support for other backends (eepp works with backends for windows and input control) is not hard given that everything is designed for it, but I don't want to maintain backends that are not useful. I'll probably have to add a new backend for SDL3 in not so long.
